client/components/application-summary.js

```diff
@@ -95,6 +98,27 @@ class ApplicationSummary extends React.Component {
     }
   }
 
+  changed = (key, subsection) => {
+    const fields = this.props.fieldsBySection[key] || [];
+    if(this.props.latest.includes(key) || this.props.latest.some(k=> fields.includes(k)) || this.props.latest.includes(subsection.repeats)) {
```

Comment:
`if (`
`k =>`
spaces please - not really your fault - this should be picked up by lint - I guess it's ignoring JSX

Reply:
actually that's a lie - this isn't a jsx file - our linting is obvs borked